s3tests_boto3/functional/test_s3.py

```diff
@@ -735,6 +1293,22 @@ def test_bucket_list_marker_empty():
     keys = _get_keys(response)
     eq(keys, key_names)
 
+@attr(resource='bucket')
+@attr(method='get')
+@attr(operation='list all keys with list-objects-v2')
+@attr(assertion='no pagination, empty continuationtoken')
+@attr('list-objects-v2')
+def test_bucket_listv2_continuationtoken_empty():
```

Comment:
@pritha-srivastava @albIN7 I don't see any tests where we are expecting back a value from `ContinuationToken` or `NextContinuationToken`. Is there a reason for this?

Reply:
Added in the modified PR.

Reply:
Could you tell me the names of them?

Reply:
test_bucket_listv2_delimiter_prefix and in test_bucket_listv2_delimiter_prefix_underscore. Is this OK?

Reply:
That's fine actually.